### PR TITLE
First draft of CloudWatch edits

### PR DESCRIPTION
**Pull request by ginny-elastic**
## What does this PR do?

From https://github.com/elastic/integrations/issues/3572:

>In https://github.com/elastic/integrations/pull/3308 we updated docs for two AWS integrations to align with the new documentation guidelines and establish the relationship between the AWS integration/package ("AWS") and integrations for individual AWS services (for example, "AWS CloudFront").
>
>Now we should update the docs for all AWS integrations for individual services to follow the same format as the updated "AWS CloudFront" integration docs.

This PR adds more context for the AWS CloudWatch integration including:

- [x] Adds context to the "Overview" including a link to the relevant AWS page and an example
- [x] Lists the types of "Data streams" for the service
- [x] "Requirements" points back to "AWS" for detailed information on credentials and permissions
- [x] "Requirements" includes any other service-specific requirements
- [x] "Setup" establishes a relationship between the AWS integration/package ("AWS") and this integration
- [x] Includes "Reference" sections

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Review by docs team 
- [ ] Review by integrations team

## Related issues

- https://github.com/elastic/integrations/issues/3572

### CODE REVIEW

File: packages/aws/_dev/build/docs/cloudwatch.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
-# cloudwatch
+# AWS CloudWatch
 
-## Logs
+## Overview
+
+The AWS CloudWatch integration allows you to monitor [AWS CloudWatch](https://aws.amazon.com/cloudwatch/). AWS CloudWatch is a monitoring and observability service built for DevOps engineers, developers, site reliability engineers (SREs), IT managers, and product owners. CloudWatch provides you with data and actionable insights to monitor your applications, respond to system-wide performance changes, and optimize resource utilization.
+
+Use the AWS CloudWatch integration to Use the AWS Cloudwatch to collect metrics and logs on the operational health of your AWS resources, applications, and services running on AWS and on-premises. Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference logs and metrics when troubleshooting an issue.
```
Comment (colleenmcginnis) [inline packages/aws/_dev/build/docs/cloudwatch.md:7]: Maybe leftover from copy and pasting from the template. 🙂 

```suggestion
Use the AWS CloudWatch integration to collect metrics and logs on the operational health of your AWS resources, applications, and services running on AWS and on-premises. Then visualize that data in Kibana, create alerts to notify you if something goes wrong, and reference logs and metrics when troubleshooting an issue.
```

File: packages/aws/_dev/build/docs/cloudwatch.md [hunk 2]
```diff
@@ -1,6 +1,46 @@
...
+
+For example, you could use the data from this integration to detect anomalous behavior in your environments. You could also use the data to troubleshoot the underlying issue by looking at additional context in the logs and metrics, such as the the source of the behavior, and more.
+
+## Data streams
+
+The AWS CloudWatch integration collects two types of data streams: logs and metrics.
+
+**Logs** help you keep a record of events happening in AWS CloudWatch.
+Log data streams collected by the AWS CloudWatch integration include The cloud account or organization id used to identify different entities in a multi-tenant environment and more. See more details in the [Logs](#logs-reference).
```
Comment (colleenmcginnis) [inline packages/aws/_dev/build/docs/cloudwatch.md:16]: Based on the documented exported fields and the example event, I _think_ the data that comes from this integration is mostly just a message from CloudWatch directly plus a lot of contextual information that comes with many different kinds of integrations. The product team can correct me if that's not right! 

```suggestion
**Logs** help you keep a record of events happening in AWS CloudWatch.
The log data stream includes the CloudWatch log message along with contextual information. See more details in the [Logs](#logs-reference).
```

File: packages/aws/_dev/build/docs/cloudwatch.md [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
+
+**Metrics** give you insight into the state of AWS CloudWatch.
+Metric data streams collected by the AWS CloudWatch integration include The cloud account name or alias used to identify different entities in a multi-tenant environment and more. See more details in the [Metrics](#metrics-reference).
```
Comment (colleenmcginnis) [inline packages/aws/_dev/build/docs/cloudwatch.md:19]:
```suggestion
**Metrics** give you insight into the state of AWS CloudWatch.
The metric data stream includes the metrics that are returned from a CloudWatch API query along with contextual information. See more details in the [Metrics](#metrics-reference).
```

File: packages/aws/_dev/build/docs/cloudwatch.md [hunk 4]
```diff
@@ -1,6 +1,46 @@
...
+
+ If you want to collect data from two or more AWS services, consider using the **AWS** integration.
+ When you configure the AWS integration, you can collect data from as many AWS services as you'd like.
+
+For step-by-step instructions on how to set up an integration, see the
+[Getting started](https://www.elastic.co/guide/en/welcome-to-elastic/current/getting-started-observability.html) guide.
+
+## Logs reference
 
 The `cloudwatch` dataset collects CloudWatch logs. Users can use Amazon
```
Comment (colleenmcginnis) [inline packages/aws/_dev/build/docs/cloudwatch.md:45]:
```suggestion
The `cloudwatch` data stream collects CloudWatch logs. Users can use Amazon 
```

**Review by colleenmcginnis — COMMENTED** [review]
Thanks @ginny-elastic! I left some suggestions below. We'll also want the development team responsible for this integration to weigh in.

File: packages/aws/_dev/build/docs/cloudwatch.md [hunk 5]
```diff
@@ -1,6 +1,46 @@
-# cloudwatch
+# AWS CloudWatch
 
-## Logs
+## Overview
+
+The AWS CloudWatch integration allows you to monitor [AWS CloudWatch](https://aws.amazon.com/cloudwatch/). AWS CloudWatch is a monitoring and observability service built for DevOps engineers, developers, site reliability engineers (SREs), IT managers, and product owners. CloudWatch provides you with data and actionable insights to monitor your applications, respond to system-wide performance changes, and optimize resource utilization.
```
Comment (bmorelli25) [inline packages/aws/_dev/build/docs/cloudwatch.md:5]: I'm concerned that this is copy/pasted from https://aws.amazon.com/cloudwatch/. Can we say this in our own words instead?

Comment (dedemorton) [reply packages/aws/_dev/build/docs/cloudwatch.md:5]: We _must_ say this in our own words. @ginny-elastic Can you let us know if you copied this text from some Elastic resource or text  provided in a spec? We need to make sure folks understand that copying text without permission from the author or company is not OK. I haven't looked at the AWS copyright statement, but I'm pretty sure this text is not fair game for our docs.

Comment (dedemorton) [reply packages/aws/_dev/build/docs/cloudwatch.md:5]: It is protected: https://aws.amazon.com/terms/

Comment (ginny-elastic) [reply packages/aws/_dev/build/docs/cloudwatch.md]: Have re-worded it.

**Comment by bmorelli25** [comment]
Replaced by https://github.com/elastic/integrations/pull/3907.

**Pull request closed without merging**